src/renderer/components/playlist-info/playlist-info.vue

```diff
@@ -19,19 +22,46 @@
     </div>
 
     <div class="playlistStats">
-      <h2 class="playlistTitle">
+      <ft-input
+        v-if="editMode"
+        ref="playlistTitleInput"
+        :placeholder="$t('User Playlists.Playlist Name')"
+        :input-tabindex="0"
+        :show-action-button="false"
+        :show-label="false"
+        :value="newTitle"
+        @input="(input) => (newTitle = input)"
+      />
+      <h2
+        v-else
+        class="playlistTitle"
+      >
         {{ title }}
       </h2>
       <p>
-        {{ videoCount }} {{ $t("Playlist.Videos") }} - <span v-if="!hideViews">{{ viewCount }} {{ $t("Playlist.Views") }} -</span>
+        {{ videoCount }} {{ $t("Playlist.Videos") }}
+        <span v-if="!hideViews && infoSource !== 'user'">
+          - {{ viewCount }} {{ $t("Playlist.Views") }}
+        </span>
+        <span>- </span>
         <span v-if="infoSource !== 'local'">
           {{ $t("Playlist.Last Updated On") }}
```

Comment:
**nitpick:** this text should be lowercase: `0 Videos - Last Updated On Nov 18, 2023` -> `0 videos - last updated on Nov 18, 2023`. Looks odd in all-caps & out of place with our other similar labels.

Reply:
Please perform case related text update for **existing** translation entries in a separate PR